### PR TITLE
scala-for-python-devs.md creating multiline string variable

### PR DESCRIPTION
**Pull request by abhi887**
Added an example for creating multiline string variable

### CODE REVIEW

File: _overviews/scala3-book/scala-for-python-devs.md [hunk 1]
```diff
@@ -136,13 +136,19 @@ These examples demonstrate how to create variables in Python and Scala.
     <tr>
       <td class="python-block">
         <code>x = 1
-        <br>x = "Hi"</code>
+        <br>x = "Hi"
+        <br>x = """foo
```
Comment (julienrf) [inline _overviews/scala3-book/scala-for-python-devs.md:140]: May I suggest to use a different name than `x` (e.g., `y`)?

Comment (abhi887) [reply _overviews/scala3-book/scala-for-python-devs.md:140]: sounds good 👍🏻

File: _overviews/scala3-book/scala-for-python-devs.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ These examples demonstrate how to create variables in Python and Scala.
     </tr>
     <tr>
       <td class="scala-block">
-        <code>val movies = Map(
+        <code>val x = Map(
```
Comment (julienrf) [inline _overviews/scala3-book/scala-for-python-devs.md:189]: Sorry to be picky, but `movies` was a better name than `x`, IMHO. If we change something here, we should rather change the Python code to use `movies` as well.

Comment (abhi887) [reply _overviews/scala3-book/scala-for-python-devs.md:189]: I thought that too, but  `x` was used throughout the documentation so I felt it looked weird.

**Review by julienrf — APPROVED** [review]

**Pull request merged**